config/server.webpack.config.dev.js

```diff
@@ -0,0 +1,37 @@
+const path = require("path");
```

Comment:
I suggest as name just `webpack.server.js`

Reply:
Would agree, but I had this because the rules for production webpack configs would be quite different. Unless we want to have them all in the same file, which could be messy 😨